recipes/newrelic/apm/java/linux.yml

```diff
@@ -318,6 +318,8 @@ install:
                   fi
                   echo $JAVA_PID >> /etc/newrelic-java/install/.ignored_pids
                 fi
+              else
```

Comment:
@mbruzina I notice we have that same echo pid to the ignore list on L319. Should we just move that to be done once on line 283 instead,?

Reply:
There are actually 3 

- 307 for PIDs matching pattern running in docker
- 319 for PIDs matching pattern not running in docker
- 321 for non-matching PIDs

as I'm typing this, I realize we can remove the if/else on the pattern match and always add the PID in question to 'ignored'...I'll fix that quickly

Reply:
@Julien4218 change is complete